SOURCES/etc/xapi.d/plugins/updater.py

```diff
@@ -125,5 +160,7 @@ def configure_logging():
 if __name__ == "__main__":
     XenAPIPlugin.dispatch({
         'check_update': check_update,
-        'update': update
+        'update': update,
+        'get_proxies': get_proxies,
```

Comment:
Shouldn't it be `get_global_proxy`?

Reply:
I think the README is outdated. Now the plan is to modify each repo's proxy configuration one by one, not yum's global proxy.

Reply:
This PR modify both the README and the plugin, I would expect them to be coherent.